Portfolio/index.html

```diff
@@ -262,153 +261,448 @@
+  }
+  .text-left b{
+    color: #9ABEE9;
+  }
+  .text-left h1{
+  
+      margin-right: 34px;
+  
+  }
+  .text-left strong{
+    font-size: 3rem;
+    color: #00D8FF;
+  }
+  
+.btn{
+    color: #004572;
+    background-color: #00D8FF;
+    
+    padding: 5px 20px;
+    border-radius: 20px;
     display: flex;
     justify-content: center;
     align-items: center;
+    margin-left: 150px;
+    cursor: pointer;
+    margin-top:10px;
 }
-.project1 img{
-    float: left;
-    /* border: 2px solid red; */
+
+
+
+  /* mobile view */
+  
+
+
+/* for under 900px */
+
+@media only screen and (max-width: 900px) {
+    .project {
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+    }
+  
+    .project img {
+      max-width: 100%;
+      order: 1;
+      margin-bottom: 20px;
+    }
+  
+    .text-right
+    {
+      max-width: 100%;
+      order: 2;
+      margin: 0;
+      margin-right: 34px;
+    }
+   
+
+    .text-left {
+      max-width: 100%;
+      order: 2;
+      margin: 0;
+  
+    }
+   
+    .project-section{
+        justify-content: center;
+    }
+   
+  }
+
+
+
+
+
+/* for under 414px */
+  @media only screen and  (max-width: 431px){
+    
+* {
+    margin: 0px;
+    padding: 0px;
+    border: 0px;
+    
+    /* h1 {
+        margin-left: 10px;
+    } */
 
 }
-.project-text1{
-    line-height: 20px;
-    margin-top: 90px;
-    margin-right: 119px;
-    float: right;
+.navbar{
+    width: 393%;
+}
+.navbar ul{
+    padding: 1px 102px;
+}
+.navbar ul li a {
+    padding: 0px 56px;
+    font-size: 1.5rem;
 }
+.navbar h1{
 
-.project2{
-    margin-left: 31px;
-    float: right;
-    position:static;
+    font-size: 4rem;
 }
-.project2 img{
-    float: right;
 
-    margin-top: -44px;
-    margin-right: 120px;
+
+.navbar img{
+    /* padding: 0px 0px; */
+    margin-left: -33px;
+   margin-top: 10px;
+   height: 28px;
+   width: 43px;
 }
-.project-text2{
-  display: contents;
-    /* border: 2px solid red; */
-    width: 90vw;
-    line-height: 20px;
-    float: left;
-    margin-top: 90px;
-    margin-left: 500px;
-}
-.project-text2 h1, b{
-    margin: 90px;
-width: 100vw;
-padding: 2px 34px;
-    display: contents;
-}
-.project3{
-    float: left;
-    position:static;
-    display: flex;
+
+.phnbtn button{
+    padding: 22px 63px;
+    margin-right:-12px;
+    width: 232px;
+}
+.phnbtn strong{
+    font-size: 0.8rem;
+}
+
+ 
+
+.gettoknowme{
+   position: relative;
+   display: inline-block;
+   margin-right: 50%;
+    justify-content: center;
+    align-items: center;
+    height: 138vh;
+    width: 404%;
 }
-.project3 img{
-    float: left;
+
+.gettoknowme p{
+    width: 54%;
+    justify-content: center;
+    align-items: center;
+    margin-top: 97px;
+    font-size: 3rem;
+    margin-left: 429px;
+}
+.name{
+    font-size: 3rem;
+        margin-left: 430px;
     
 }
+.gettoknowme h1{
+    margin-left: 577px;
+    width: 585px;
+    font-size: 5rem;
+    align-self: center;

```

Comment:
The closing tag for '<h1>' within the '.abtme' section should come after the '<img>' tag.